public/index.html

```diff
@@ -8,8 +8,12 @@
     manifest.json provides metadata used when your web app is added to the
     homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
   -->
-  <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-  <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+  <link rel="icon" type="image/png" href="/favicon/favicon-48x48.png" sizes="48x48" />
```

Comment:
You need to add %PUBLIC_URL%/ as prefix to these paths

Reply:
added %PUBLIC_URL%/ as prefix to these paths #2